projects/bulma-app/src/assets/docs/navbar/header-description.md

```diff
@@ -2,13 +2,29 @@
 
 The `navbar` component is a responsive and versatile horizontal navigation bar with the following structure.
 
-- navbar the main container
-  - `navbar-brand` the left side **always visible.**, which usually contains the logo and optionally some links or icons
-    - `navbar-burger` the hamburger icon, which toggles the navbar menu on touch devices
-  - `navbar-menu` the right side, hidden on touch devices, visible on desktop
-    - `navbar-start` the left part of the menu, which appears next to the navbar brand on desktop
-    - `navbar-end` the right part of the menu, which appears at the end of the navbar
-      - `navbar-item` each single item of the navbar, which can either be an a or a div
-        - `navbar-link` a link as the sibling of a dropdown, with an arrow
-        - `navbar-dropdown` the dropdown menu, which can include navbar items and dividers
-          - `navbar-divider` a horizontal line to separate navbar items
+To display a Navbar component in your app and verify that everything works.
+You need to import the `BulmaNavbarModule` that you want to display by adding the following lines to your `app.module.ts` file.
+
+```javascript
+import { BulmaNavbarModule} from 'ngx-bulma'
+ @NgModule({
+   imports: [BulmaNavbarModule]
+ });
+```
+
+The most basic Modal needs only of `<bu-nav>` element with some content. However, Bulma components provides a number of preset sections that you can use inside of `</bu-nav>`
+
+| Element             | Description                                                                        |     |     |     |
+| ------------------- | ---------------------------------------------------------------------------------- | --- | --- | --- |
+| `<bu-nav>`          | Parent component                                                                   |     |     |     |
+| `<bu-nav-brand>`    | the left side **always visible.**, which usually contains the logo and optionally. |
+| `<bu-navbar-start>` | the left part of the menu, which appears next to the navbar brand on desktop.      |
+| `<bu-navbar-end>`   | the right part of the menu, which appears at the end of the navbar.                |
+
+| Propert           | Description                                                        |     |     |     |
```

Comment:
The spelling is wrong. It should be "Properties" instead of "Propert"

Reply:
this  one  already changed, why it is showing here 
![des](https://user-images.githubusercontent.com/20164301/83133122-025de000-a100-11ea-85c8-5ad7b8a17476.png)